src/components/SideBar.js

```diff
@@ -0,0 +1,83 @@
+import { Drawer, Box, Toolbar, Typography, Divider, Avatar} from "@mui/material"
+import logo from '../assets/images/logoo.png'
+import { NavLink } from 'react-router-dom'
+import "./style.css"
+
+export const SideBar = ({ drawerWidth = 255 }) => {
+  return (
+    <Box
+      component='nav'
+      sx={{ width: { sm: drawerWidth }, flexShrink: { sm: 0 } }}
+    >
+      <Drawer
+        variant="permanent"
+        open={false}
+        sx={{
+          display: { xs: 'block' },
+          '& .MuiDrawer-paper': {
+            boxSizing: 'border-box',
+            width: drawerWidth,
+            backgroundColor: '#363740'
+          }
+        }}
+      >
+        <Toolbar>
+          <Box
+            component='div'
+
+            sx={{
+              width: '100%',
+              paddingTop: '20px',
+              display: 'flex',
+              justifyContent: 'space-evenly',
+              alignItems: 'center'
+            }}
+          >
+            <Avatar alt="logo" src={logo} />
+            <Typography sx={{
+              color: '#A4A6B3',
+              fontWeight: '600',
+              fontFamily: ['Mulish', 'sans-serif'].join(),
+            }} variant="h6">Plagio Control</Typography>
+          </Box>
+
+
+
+        </Toolbar>
+        {/* <Divider /> */}
+        <br />
+        <br />
+        <div style={{ width: '100%', height: '100vh' }}>
+          <nav>
+            <ul className="sidebar-menu">
+              <li >
+                <NavLink to='/'><i className="fas fa-chart-pie"></i>Inicio</NavLink>
+              </li>
+              <li >
+                <NavLink to='/trabajos'><i className="fas fa-briefcase"></i>Trabajos</NavLink>
```

Comment:
this code should be in a json file.
`
const navigation = [
      {label: 'Inicio', uri: '/inicio'},
      {label: 'Trabajos', uri: '/trabajo'},
]
`
and you should mapping the json in the DOM